tests/functional/spec/features/routing/number.spec.js

```diff
@@ -57,66 +57,54 @@ describe("Feature: Routing on a Number", () => {
         expect(browser.getUrl()).to.contain(CorrectAnswerPage.pageName);
       });
 
-      it("When I enter a smaller number, Then I should be routed to the incorrect page", () => {
+      it("When I enter a number equal to 123, Then I should be routed to the incorrect page", () => {
         $(CorrectAnswerPage.previous()).click();
-        $(NumberQuestionPage.answer()).setValue(2);
+        $(NumberQuestionPage.answer()).setValue(123);
         $(NumberQuestionPage.submit()).click();
         expect(browser.getUrl()).to.contain(IncorrectAnswerPage.pageName);
       });
-    });
-  });
-
-  describe("Less Than", () => {
-    describe("Given I start number routing less then survey", () => {
-      before(() => {
-        browser.openQuestionnaire("test_routing_number_less_than.json");
-      });
-
-      it("When I enter a smaller number, Then I should be routed to the correct page", () => {
-        $(NumberQuestionPage.answer()).setValue(77);
-        $(NumberQuestionPage.submit()).click();
-        expect(browser.getUrl()).to.contain(CorrectAnswerPage.pageName);
-      });
 
-      it("When I enter a greater number, Then I should be routed to the incorrect page", () => {
-        $(CorrectAnswerPage.previous()).click();
-        $(NumberQuestionPage.answer()).setValue(765);
+      it("When I enter a smaller number, Then I should be routed to the incorrect page", () => {
+        $(IncorrectAnswerPage.previous()).click();
+        $(NumberQuestionPage.answer()).setValue(2);
         $(NumberQuestionPage.submit()).click();
         expect(browser.getUrl()).to.contain(IncorrectAnswerPage.pageName);
       });
     });
   });
 
-  describe("Greater Than or Equal", () => {
-    describe("Given I start number routing greater then or equal survey", () => {
+  describe("Less Than", () => {
+    describe("Given I start number routing less than survey", () => {
       before(() => {
-        browser.openQuestionnaire("test_routing_number_greater_than_or_equal.json");
+        browser.openQuestionnaire("test_new_routing_number_less_than.json");
       });
 
-      it("When I enter a greater number, Then I should be routed to the correct page", () => {
-        $(NumberQuestionPage.answer()).setValue(555);
+      it("When I enter a smaller number, Then I should be routed to the correct page", () => {
```

Comment:
We should make the tests explicit and consistent. For example, `When I enter a number less than 123, Then ...`

Some tests already do this, so can we just make sure it is consistent between these files.